src/watchful.ts

```diff
@@ -86,15 +93,18 @@ export class Watchful extends Construct implements IWatchful {
       );
     }
 
-    this.dash = new cloudwatch.Dashboard(this, 'Dashboard', { dashboardName: props.dashboardName });
+    if (!props.noDashboard){
+      this.dash = new cloudwatch.Dashboard(this, 'Dashboard', { dashboardName: props.dashboardName });
```

Comment:
the dash property in this class should be explicitly marked as undefined (`private dash?: cloudwatch.Dashboard`)